lib/src/simple_op_store.rs

```diff
@@ -102,67 +100,28 @@ fn upgrade_to_thrift(store_path: PathBuf) -> std::io::Result<()> {
         }
     }
 
-    fs::write(tmp_store_path.join("thrift_store"), "")?;
     let backup_store_path = store_path.parent().unwrap().join("op_store_old");
+    // Delete existing backup (probably from an earlier upgrade to Thrift)
+    fs::remove_dir_all(&backup_store_path).ok();
     fs::rename(&store_path, backup_store_path)?;
     fs::rename(&tmp_store_path, &store_path)?;
 
```

Comment:
This `remove_dir_all()` looked scary at first, but seems fine. If the previous upgrade failed after `mv store_path backup_store_path`, the next upgrade wouldn't reach here.